config/manager/operator-envs

```diff
@@ -14,3 +14,4 @@ CONCURRENCY_VERTICARESTOREPOINTSQUERY
 CONCURRENCY_VERTICASCRUTINIZE
 CONCURRENCY_SANDBOXCONFIGMAP
 CONCURRENCY_VERTICAREPLICATOR
+BROADCASTER_BURST_SIZE
```

Comment:
Sorry I forgot to tell you. You also need to add this to `template-helm-chart.sh`. Also need to add a new parameter to values.yaml. Look at `WEBHOOKS_ENABLED`(line 227 in template-helm-chart.sh)  as an example and do the same for `BROADCASTER_BURST_SIZE`

Reply:
No worries. I added a new commit to address that. You can take a look.  I built locally and deployed to my local kind cluster. The burst size is 100. No external environmental variable was used.